### DISCUSSION TITLE
Getting Started I can't download de program con mi Mac M2 Pro with Sequoia 15.0

### DISCUSSION THREAD

**Discussion by montilibi** [Homebrew/discussions#5664] [Getting started]
### Output of `brew config`

```shell
.
```

### Output of `brew doctor`

```shell
.
```

### Description of issue

I am installing the Homebrew program ( specifically The Missing Package Manager) for macOS on mi Mac. First I open Terminal and y copy the command on the page. Afterwards it starts downloading, when it is about to finish, it appears on the code that there are errors and it forces origin, which leads me to the first step (copying again the command). I have tried a few times more and the problem repeats. I am going to upload a photo so someone can help me with the problem
It is urgent.
Thank you

<img width="787" alt="Captura de pantalla 2024-10-10 a las 10 54 13" src="https://github.com/user-attachments/assets/40383c65-afd2-48d1-8476-fc3f2babaff2">

**Comment by gromgit** [comment]
https://docs.brew.sh/Common-Issues#installation-fails-with-unknown-revision-or-path-not-in-the-working-tree
